Source/effects.cpp

```diff
@@ -1084,6 +1084,11 @@ void stream_stop()
 static void stream_play(TSFX *pSFX, int lVolume, int lPan)
 {
 	BOOL success;
+#ifndef DISABLE_STREAMING_SOUNDS

```

Comment:
Have you considered making this an option on all platforms? Aren't there any scenarios where streaming could be problematic and someone could want to turn it off?

Reply:
Streaming should not be problematic elsewhere

Reply:
The flag isn't platform specific though. It's an option in the the main cmake file.

Reply:
Yeah I know... was just wondering about the overhead of introducing `#ifdef`s at all... I really dislike those. Feels to me that it would be preferred to have as much of the codebase as possible not relying on those but on soft settings instead.

Reply:
For settings I agree but this shouldn't be a setting - it's a workaround against a broken library platform library implementation and we shouldn't carry around this code to other platforms.